runtime/actions/query.go

```diff
@@ -131,6 +145,23 @@ func (o *QueryOperand) IsNull() bool {
 func (o *QueryOperand) toSqlOperandString(query *QueryBuilder) string {
 	switch {
 	case o.IsValue() && !o.IsArrayValue():
+		if o.IsTimePeriodValue() {
+			tp, _ := o.value.(timeperiod.TimePeriod)
+			sql := "NOW()"
+			if tp.IsTimezoneRelative() && query.timezone != "" {
+				sql = fmt.Sprintf("(NOW() AT TIME ZONE '%s')", query.timezone)
+			}
+			if tp.Offset != 0 {
+				sql = fmt.Sprintf("%s + INTERVAL '%d %s'", sql, tp.Offset, tp.Period)
+			}
+			if tp.Complete {
+				sql = fmt.Sprintf("date_trunc('%s', %s)", tp.Period, sql)
```

Comment:
```suggestion
				sql = fmt.Sprintf("DATE_TRUNC('%s', %s)", tp.Period, sql)
```

I think we're currently using caps for the runtime sql gen, so best to be consistent